static/js/portico/markdown_panel.js

```diff
@@ -0,0 +1,208 @@
+import * as diff from "diff";
+import * as diff2html from "diff2html";
+import hljs from "highlight.js/lib/core";
+import xml from "highlight.js/lib/languages/xml";
+import $ from "jquery";
+
+import * as channel from "../channel";
+import * as markdown from "../markdown";
+// Importing stream_data to avoid import error
```

Comment:
While importing `get_helpers` from `markdown_config.js`.
I am facing the following error,
```bash
Uncaught TypeError: _filter__WEBPACK_IMPORTED_MODULE_25__.Filter is not a constructor
    at new MessageListData (app~archive~markdown-panel.js:20365)
    at Module../static/js/all_messages_data.js (app~archive~markdown-panel.js:1436)
    at __webpack_require__ (markdown-panel.js:854)
    at fn (markdown-panel.js:151)
    at Module../static/js/stream_topic_history.js (app~archive~markdown-panel.js:51712)
    at __webpack_require__ (markdown-panel.js:854)
    at fn (markdown-panel.js:151)
    at Module../static/js/stream_data.js (app~archive~markdown-panel.js:47430)
    at __webpack_require__ (markdown-panel.js:854)
    at fn (markdown-panel.js:151)
    at Module../static/js/filter.js (app~archive~markdown-panel.js:11094)
    at __webpack_require__ (markdown-panel.js:854)
    at fn (markdown-panel.js:151)
    at Module../static/js/narrow_state.js (app~archive~markdown-panel.js:25623)
    at __webpack_require__ (markdown-panel.js:854)
    at fn (markdown-panel.js:151)
    at Module../static/js/hash_util.js (app~archive~markdown-panel.js:13270)
    at __webpack_require__ (markdown-panel.js:854)
    at fn (markdown-panel.js:151)
    at Module../static/js/markdown_config.js (markdown-panel.js:1627)
    at __webpack_require__ (markdown-panel.js:854)
    at fn (markdown-panel.js:151)
    at Module../static/js/portico/markdown_panel.js (markdown-panel.js:1764)
    at __webpack_require__ (markdown-panel.js:854)
    at fn (markdown-panel.js:151)
    at Object.41 (markdown-panel.js:2008)
    at __webpack_require__ (markdown-panel.js:854)
    at checkDeferredModules (markdown-panel.js:46)
    at markdown-panel.js:1590
    at markdown-panel.js:1593
```

I am currently importing `stream_data.js` to avoid this.

Can someone help me with this?